LESSONS.md

```
@@ -67,6 +67,15 @@ See `AGENTS.md` guardrail #10 for the self-improvement loop process.
 
 ---
 
+## Email Personalization
+
+### Outbound greetings must fail safe instead of guessing names
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)